documentation/00_getting_started/00-getting-started.html.md

```diff
@@ -9,7 +9,9 @@ Getting started requires three steps:
 
 1. [Install OpenFL and Haxe](http://www.openfl.org/download/)
 2. [Install HaxeFlixel](/documentation/install-haxeflixel)
-4. [HaxeFlixel Hello World](/documentation/hello-world-command-line)
+3. [HaxeFlixel Hello World](/documentation/hello-world-command-line)
+
+For FlashDevelop users: [Getting started easily with FlashDevelop](/documentation/FlashDevelop/)
```

Comment:
This makes no mention of HaxeDevelop, which some users might not know it's the same as FlashDevelop. Maybe we should put ```FlashDevelo / HaxeDevelop```?